src/components/Example/index.tsx

```diff
@@ -135,8 +136,8 @@ export default function Example() {
       await useScript(packages);
 
       if (exampleId) {
-        fetchDocDataById(exampleId).then((res) => {
-          const code = transformTsToJs(res?.content || '', packageGlobals);
+        fetchDocDataById(exampleId).then(async(res) => {
+          const code = await transformTsToJs(res?.content || '', packageGlobals);
```

Comment:
Using `??` instead of `||`  maybe more robust.

Reply:
Fixed.